spec/mem/tree_borrows/memory.md

```diff
@@ -234,11 +344,14 @@ impl<T: Target> Memory for TreeBorrowsMemory<T> {
         ptr: Pointer<Self::Provenance>,
         ptr_type: PtrType,
         fn_entry: bool,
-        size_computer: impl Fn(LayoutStrategy, Option<PointerMeta<Self::Provenance>>) -> Size,
+        _size_computer: impl Fn(LayoutStrategy, Option<PointerMeta<Self::Provenance>>) -> Size, // TODO: Remove this later?
```

Comment:
yes. The closure passed there is always the same, you can "inline" this to where it's actually needed.